### PR TITLE
resolving ui issues

### PR DESCRIPTION
**Pull request by Mehradml**
The resolved issues are as follows:
- https://github.com/project-chip/zap/issues/287
- https://github.com/project-chip/zap/issues/282
- https://github.com/project-chip/zap/issues/271
- https://github.com/project-chip/zap/issues/272
- https://github.com/project-chip/zap/issues/281
- https://github.com/project-chip/zap/issues/283
- https://github.com/project-chip/zap/issues/284
- https://github.com/project-chip/zap/issues/285
- https://github.com/project-chip/zap/issues/288
- https://github.com/project-chip/zap/issues/290
- https://github.com/project-chip/zap/issues/291
- https://github.com/project-chip/zap/issues/292
- https://github.com/project-chip/zap/issues/293
- https://github.com/project-chip/zap/issues/298

### CODE REVIEW

File: src/boot/ws.js [hunk 1]
```diff
@@ -104,17 +104,21 @@ onWebSocket(dbEnum.wsCategory.init, (data) =>
   console.log(`Init message received: ${data}`)
 )
 
-onWebSocket(dbEnum.wsCategory.tick, (data) =>
-  console.log(`Tick received: ${data}`)
-)
```
Comment (brdandu) [inline src/boot/ws.js:109]: Can we tie these to some debug flag so we automatically get these when we turn on the debug level. I remeber Timotej did something like this. Same goes for all the other commented out code in this file

File: src/components/ZclAttributeReportingManager.vue [hunk 2]
```diff
@@ -220,6 +332,18 @@ export default {
     isRowDisabled(attributeId) {
       return !this.editableAttributesReporting[attributeId]
     },
+    displayAttrWarning(row) {
+      // TODO display proper warnings
```
Comment (brdandu) [inline src/components/ZclAttributeReportingManager.vue:336]: Is there some todo here remaining?

File: src/components/ZclCommandManager.vue [hunk 3]
```diff
@@ -185,7 +187,13 @@ export default {
...
                 (this.selectionServers.includes(this.selectedCluster.id) &&
                   row.source == 'server'))
-      && !this.selectionOut.includes(this.hashCommandIdClusterId(row.id, this.selectedCluster.id))
+      && !this.selectionOut.includes(this.hashCommandIdClusterId(row.id, this.selectedCluster.id)) ||
+        this.isCommandRequired(row) && (
+        (this.selectionClients.includes(this.selectedCluster.id) &&
+                  row.source == 'server') ||
+                (this.selectionServers.includes(this.selectedCluster.id) &&
+                  row.source == 'client'))
+      && !this.selectionIn.includes(this.hashCommandIdClusterId(row.id, this.selectedCluster.id))
```
Comment (brdandu) [inline src/components/ZclCommandManager.vue:196]: What is this above change doing?

File: src/boot/ws.js (hunk 1 above)
Comment (tecimovic) [reply src/boot/ws.js:109]: We have a good logging on back-end tied to loglevels. I don't even know how this is done on front-end.

Would it make sense for back-end to control this? 

Or does it make more sense to this fully in front-end?

Comment (tecimovic) [reply src/boot/ws.js:109]: BTW: This is a cleanup. We should not hold up the PR for this.

**Comment by tecimovic** [comment]
https://github.com/project-chip/zap/issues/287: See my comment on the bug. There is a weird jump.
https://github.com/project-chip/zap/issues/282: Same problem as above. When table is too small, there is a jump. (I noticed that the jump doesn't happen if the scrollbar is not visible, so jump is only if the table is long enough to cause a scrollbar.)

**Review by tecimovic — APPROVED** [review]

File: src/components/ZclCommandManager.vue (hunk 3 above)
Comment (tecimovic) [reply src/components/ZclCommandManager.vue:196]: One of the bugs was that the command handling needs the warnings if you disable mandatory command. I don't know what the code does, but I clicked around there and it worked for me. So whatever the code does, is not doing too much damage. :)

File: src/boot/ws.js (hunk 1 above)
Comment (brdandu) [reply src/boot/ws.js:109]: My main concern was that if that was added by someone for some error monitoring. So I was wondering if we will need that again. May be have the front end show this for now. We could come up with a way to show all issue in one place later?

**Review by bzbarsky-apple — APPROVED** [review]
Rubberstamping, generally....

**Pull request merged**